### PR TITLE
Add some missing docs for the web server bundle

### PR DESCRIPTION
**Pull request by fabpot**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

This is an alternative to #22471

### CODE REVIEW

File: src/Symfony/Bundle/WebServerBundle/Command/ServerStartCommand.php [hunk 1]
```diff
@@ -53,10 +53,15 @@ protected function configure()
...
             ->setHelp(<<<'EOF'
-The <info>%command.name%</info> runs a local web server:
+<info>%command.name%</info> runs a local web server: By default, the server
+listens on <comment>127.0.0.1</> address and the port number is automatically selected
+as the first free port starting from <comment>8000</>:
 
   <info>php %command.full_name%</info>
 
+The server is run in the background and you can keep executing other commands.
+Execute <comment>server:stop</> tp stop it.
```
Comment (ro0NL) [inline src/Symfony/Bundle/WebServerBundle/Command/ServerStartCommand.php]: `to stop it`

File: src/Symfony/Bundle/WebServerBundle/Command/ServerStatusCommand.php [hunk 2]
```diff
@@ -38,6 +38,20 @@ protected function configure()
                 new InputOption('filter', null, InputOption::VALUE_REQUIRED, 'The value to display (one of port, host, or address)'),
             ))
             ->setDescription('Outputs the status of the local web server for the given address')
+            ->setHelp(<<<'EOF'
+<info>%command.name%</info> shows the details of the given local web
+server, such as the address and port where is listening to:
```
Comment (ro0NL) [inline src/Symfony/Bundle/WebServerBundle/Command/ServerStatusCommand.php]: `it's listening to`

**Review by ro0NL — APPROVED** [review]

**Review by nicolas-grekas — APPROVED** [review]
:+1:

**Review by stof — APPROVED** [review]

File: src/Symfony/Bundle/WebServerBundle/Command/ServerLogCommand.php [hunk 3]
```diff
@@ -47,11 +47,24 @@ protected function configure()
...
+messages generated by your application:
+
+  <info>php %command.full_name%</info>
+
+To get the information as a machine readable format, use the
+<comment>--filter</> option:
+
+<info>php %command.full_name% --filter=port</info>
+
+Supported values are <comment>port</>, <comment>host</>, and <comment>address</>.
```
Comment (stof) [inline src/Symfony/Bundle/WebServerBundle/Command/ServerLogCommand.php]: that's wrong actually. `filter` is about filtering logs with the ExpressionLanguage

**Review by stof — CHANGES_REQUESTED** [review]

File: src/Symfony/Bundle/WebServerBundle/Command/ServerLogCommand.php (hunk 3 above)
Comment (fabpot) [inline src/Symfony/Bundle/WebServerBundle/Command/ServerLogCommand.php]: bad copy/paste, fixed

File: src/Symfony/Bundle/WebServerBundle/Command/ServerStartCommand.php (hunk 1 above)
Comment (fabpot) [inline src/Symfony/Bundle/WebServerBundle/Command/ServerStartCommand.php]: fixed

File: src/Symfony/Bundle/WebServerBundle/Command/ServerStatusCommand.php (hunk 2 above)
Comment (fabpot) [inline src/Symfony/Bundle/WebServerBundle/Command/ServerStatusCommand.php]: fixed

File: src/Symfony/Bundle/WebServerBundle/Command/ServerLogCommand.php [hunk 4]
```diff
@@ -47,11 +47,22 @@ protected function configure()
...
+messages generated by your application:
+
+  <info>php %command.full_name%</info>
+
+To get the information as a machine readable format, use the
+<comment>--filter</> option:
+
+<info>php %command.full_name% --filter=port</info>
+EOF
         ;
```
Comment (yceruto) [inline src/Symfony/Bundle/WebServerBundle/Command/ServerLogCommand.php]: missing `)` fixed in https://github.com/symfony/symfony/pull/22626

**Pull request merged**